benchmark/scenarios/write_request_load.py

```diff
@@ -0,0 +1,289 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Read request load scenario for the control service benchmarks.
+"""
+from collections import deque
+from itertools import repeat
+
+from zope.interface import implementer
+import eliot
+
+from twisted.internet.defer import CancelledError, Deferred, succeed
+from twisted.internet.task import LoopingCall
+
+from flocker.common import loop_until, timeout
+
+from .._interfaces import IScenario
+
+DEFAULT_SAMPLE_SIZE = 5
+
+# XXX make it a common class for Read and Write scenarios.
+# Note that the docstrings are not up-to-date because they are being updated
+# in the Read scenario
+class WRateMeasurer(object):
+    """
+    Measures the rate of requests in requests per second
```

Comment:
The W is temporary. The rate measurer is exactly the same one for Read and Write requests. I will take it out and put it in a common file for all the scenarios. But right now I need different names so I can import it in the tests